README.md

```diff
@@ -48,7 +48,7 @@ pod 'ResilientDecoding', '~> 0.9'
 
 ## Decoding
 
-The main interface to this package is the `@Resilient` property wrapper. It can be applied to three kinds of properties: `Optional`,  `Array`, and custom types conforming to the `ResilientRawRepresentable` protocol that this package provides. 
+The main interface to this package is the `@Resilient` property wrapper. It can be applied to four kinds of properties: `Optional`,  `Array`,  `Dictionary`, and custom types conforming to the `ResilientRawRepresentable` protocol that this package provides. 
```

Comment:
Adding this here so it's in my review and not lost. Can you clarify this comment? https://github.com/airbnb/ResilientDecoding/pull/21#discussion_r414299488
It seems based on your updates that we're testing the "right" thing now?
I guess I'm curious how the array tests come into play here.

Reply:
Yes. In earlier versions of this library (not open source) the behavior was that non-optional arrays actually reported errors if the key was missing or the value was null. As a result, the tests that expected no errors needed valid values for the fields they weren't interested in. A survey of use-cases revealed almost no one actually wanted this behavior, so it was simplified to behave like an optional, where a missing key or null value would just silently fall back to `[]`. This was never cleaned up in the tests, and the dictionary tests are just copy-pasta from the array tests.

Reply:
Thanks. Maybe at some point we can simplify.